### DISCUSSION TITLE
Week 16 Report

### DISCUSSION THREAD

**Discussion by derrrobin** [SE-TINF22B6/CookPit#99] [General]
# Cookpit Team Weekly Blog: Update on Bug Fixes and Search Functionality

Welcome to this week’s blog post! The Cookpit team has been hard at work focusing on improving the search functionality and addressing bugs. Our current search system relies on an API, which we are considering changing for better efficiency and performance. An other mayor point to mention is, that now loading images from our database is working. Alongside these technical improvements, we had a productive retrospective meeting. Here’s a summary of our key takeaways from the retrospective, categorized into four main areas: Continue, Stop, Invent, and Act.

### Continue
 - **Trust and Team Spirit:** Our progress was significantly bolstered by the strong trust and spirit within the team.
-  **Motivation and Fun:** The motivation to develop an outstanding product, combined with having fun at work, enhanced our productivity.
![Continue](https://github.com/SE-TINF22B6/CookPit/assets/123726577/9269fd47-500c-4ccd-8f08-634e40b59fe9)

### Stop
- **Documentation and Diagrams:** The effort required for maintaining documentation and diagrams was seen as a hindrance.
- **Errors and Misunderstandings:** Small misunderstandings and frequent errors were identified as slowing our progress.
-  **Stress and Overload:** High stress levels and the overload from meetings and other obligations impeded our workflow.
-  **Database Problems:** Issues with the database caused significant delays and frustrations.
![Stop](https://github.com/SE-TINF22B6/CookPit/assets/123726577/9b691179-3163-4343-a9b7-b148346cefc4)

### Invent
- **More Efficient Documentation:** We need to find ways to streamline the documentation process to make it less time-consuming.
- **Better Error Culture:** Developing a culture that quickly identifies and addresses errors effectively is essential.

![Invent](https://github.com/SE-TINF22B6/CookPit/assets/123726577/8a7d77a3-f6bd-456b-b53e-6937f5cb601f)

### Act
- **Define Clear Tasks:** Clearly define tasks for the next two weeks to ensure focused progress before the demo.
- **Focus on Important Stuff:** Concentrate on the most critical tasks to maximize impact.
![Act](https://github.com/SE-TINF22B6/CookPit/assets/123726577/6a3e38d5-6ab4-4cce-8967-70821d5a44f8)

By focusing on these key areas, we aim to enhance our team’s efficiency and productivity, ensuring we deliver a high-quality product. Stay tuned for our last updates next week!#

Our full summary of the retro you can finde here: https://github.com/SE-TINF22B6/CookPit/blob/main/Docs/CookPitTeamRetrospective.md

**Comment by tech-n1c** [comment]
Dear CookPit-Team, 

it looks like you had quite the productive retro session for your project! I'm sure you will learn a lot from this technique and it will help you accomplish the project this semester! Looking back on what you've done, what went good/wrong is key to improvement and steady progress. 

Good job on getting your API working, I can't wait to see the final result in your demo!

Sincerely
team time2shine

**Comment by BenjaminB109** [comment]
Dear CookPit Team,
It's nice that the pictures are now loading from the database.
You have given a very good summary of the Retrospective. I'm looking forward to the high quality product.👀

**Comment by TadaXD20** [comment]
Dear CookPit team,
your retrospective board is very detailled and I see that you put a lot of thought in it. Your hard work is really paying off because it looks like you are well on schedule 👍🏻 I am thrilled to see that you are already so close to the finish line ^^
Best regards from the Plapy team
